week-2/Homework/extra/4-radio-stations.js

```diff
@@ -15,18 +15,56 @@
 
 // `getAllFrequencies` goes here
 
+let arr=[];
```

Comment:
Do you think this should be in the global scope? If I had another function, it might be able to mutate this, which could cause bugs. Where might it better live?